src/rfac/rfrdexpt.c

```
@@ -0,0 +1,177 @@
+  }
+
+  if(iv->equidist == true)
+  {
+    CONTROL_MSG(CONTROL, "expt. IV curve is equidistant\n");
+  }
+  else
+  {
+    CONTROL_MSG(CONTROL, "expt. IV curve is not equidistant\n");
+  }
+
+  if (!iv) 
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)